pkg/bgpv1/agent/controller.go

```diff
@@ -352,46 +353,9 @@ func (c *Controller) Reconcile(ctx context.Context) error {
 		return fmt.Errorf("invalid BGP peering policy %s: %w", policy.Name, err)
 	}
 
-	// parse any virtual router specific attributes defined on this node via
```

Comment:
This whoel block goes away, since all this information can be gained just from looking at 'localNode'